apisix/plugins/limit-count/init.lua

```diff
@@ -302,7 +309,8 @@ function _M.rate_limit(conf, ctx)
 
     if conf.show_limit_quota_header then
         core.response.set_header("X-RateLimit-Limit", conf.count,
-            "X-RateLimit-Remaining", remaining)
+                "X-RateLimit-Remaining", remaining,
```

Comment:
Why change 4-spaces indentation to 8-spaces here?

Reply:
fixed